docs/agent/config/v3.mdx

```diff
@@ -2,435 +2,106 @@
 title: "Version 3"
 ---
 
-import AgentConfigurations from "./_agent-configurations.mdx";
-
 ## Breaking Changes
 
-The `agent:` key is now **required** in the v3 configuration file as a parent field, and [agent specific fields](/docs/agent/config/v3/#agent-configuration) are nested underneath.
+- [Agent specific fields](/docs/agent/config/v3/#agent-configuration) are no longer top level-keys, but instead nested under the **required** `agent:` field.
+- There are now _4_ top-level keys (`version`, `agent`, `endpoints`, and `tunnels`.)
+- `tunnels` are deprecated in favor of `endpoints`.
```

Comment:
@tammybailey  Got these included for you (Thanks for the reminder) 🖖 Happy to wordsmith / alter it more if needed.